k8s/Dockerfile

```
@@ -7,7 +7,13 @@ FROM base-${TARGETARCH}
 ARG DEBIAN_FRONTEND=noninteractive
 
 # install all of the tools we need.
-RUN apt-get update && apt-get upgrade -y && apt-get install nginx vim emacs libmariadb-dev-compat default-libmysqlclient-dev libssl-dev nodejs npm python3-opencv libgl1 tesseract-ocr nano nfs-common sudo iputils-ping hylafax-client pandoc texlive texlive-luatex texlive-latex-extra texlive-xetex postgresql ca-certificates curl libpq-dev wkhtmltopdf -y
+RUN apt-get update && \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)